packages/@react-aria/radio/src/useRadioGroup.ts

```diff
@@ -63,7 +63,6 @@ export function useRadioGroup(props: AriaRadioGroupProps, state: RadioGroupState
   });
 
   let onKeyDown = (e) => {
-    e.preventDefault();
     let walker = getFocusableTreeWalker(e.currentTarget, {from: e.target});
```

Comment:
Can we create this after the switch? Not sure the perf impact of creating one of these on every key press, even if we don't need it.